External/FEXCore/include/FEXCore/IR/IREmitter.h

```diff
@@ -19,11 +19,37 @@ friend class FEXCore::IR::Pass;
 friend class FEXCore::IR::PassManager;
 
   public:
-    IREmitter()
-      : DualListData {8 * 1024 * 1024} {
+    IREmitter(FEXCore::Utils::ThreadPoolAllocator &ThreadAllocator)
+      : DualListData {ThreadAllocator, 8 * 1024 * 1024} {
       ResetWorkingList();
     }
 
+    void ClaimBuffer() {
+      DualListData.ClaimBuffer();
+    }
+    void UnclaimBuffer() {
+      auto Now = std::chrono::high_resolution_clock::now();
```

Comment:
Would a `steady_clock` be sufficient here as well? A high resolution clock doesn't appear necessary given that the reclamation interval is 5 seconds, and `steady_clock` (being a monotonic clock) will have more predictable behavior on system clock changes. (You wouldn't want reclamation to stop working for an hour because the system entered DST!)

Reply:
You right. Resolved.